src/main/java/com/bookingapp/dto/user/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,24 @@
+package com.bookingapp.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import org.hibernate.validator.constraints.Length;
+
+public record UserRegistrationRequestDto(
+        @NotBlank
+        @Email
+        String email,
+        @Length(min = 4, max = 50)
+        @NotBlank
+        String password,
```

Comment:
@NotNull
        @Pattern(
                regexp = "^[a-zA-Z0-9]{6,32}$",
                message = "Password must contain at least one digit and letter"
        )